api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/EligibilityOverrideTestDataFactory.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.dhsc.htbhf.claimant.testsupport;
+
+import uk.gov.dhsc.htbhf.claimant.entity.EligibilityOverride;
+import uk.gov.dhsc.htbhf.dwp.model.EligibilityOutcome;
+
+public class EligibilityOverrideTestDataFactory {
+
+    public static EligibilityOverride eligibilityConfirmedWithNoChildrenForFiveYears() {
+        return EligibilityOverride.builder()
+                .eligibilityOutcome(EligibilityOutcome.CONFIRMED)
+                .build();
+    }
+
+    public static EligibilityOverride eligibilityNotConfirmed() {
```

Comment:
I think this would be better as `aNotConfirmedEligibilityOverride`, this method name sounds more like an EligibilityOutcome that is being created.